### DISCUSSION TITLE
What you guys think of Herd? Will it replace laragon?

### DISCUSSION THREAD

**Discussion by notjustanothergithubuser** [leokhoa/laragon#741] [General]
first of all i am a big fan of laragon but today i tried herd and i love the ui bud sadly some must have futures are paid which will make me staying at laragon. What you guys think about Herd? Will you move to Herd or staying at laragon?

**Comment by Pen-y-Fan** [comment]
I haven't tried it on Windows, as you say with MySQL being a paid (pro) option, unless you only use SQLite, saying on Laragon is a no-brainer.

On my Mac, I use Valet and PHP Mon, which works well. I tried to install Hurd, which is supposed to detect Valet and import the sites. Pretty URLs should be auto-generated, but it didn't work, I suspect there is a conflict with DNSmasq, it would not start in Hurd, and even after uninstalling the original version, installed by Valet, it would start in Herd, pretty URLs would still not work.

**Comment by RobiNN1** [comment]
It never replace Laragon, mainly because of mysql and maybe redis.

It doesn't have Apache, which is still used by many developers in production.
No Memcache.
Its impossible to install older PHP versions, sometimes you need 5.6 when old project requires it.

For dns Herd uses also windows host file.

Other Herd features such as php per site can be achieved manually if you know how to do that.

For mail you can use https://github.com/axllent/mailpit.

**Comment by achmadhadikurnia** [comment]
Hello.
The only i loved on Herd is can use php version per site. so i move to Herd. but what I feel is that the site running on Herd feels slow. I don't know if this is because the laptop is my responsibility or what, because I moved at the same time as I reinstalled Windows.

**Comment by mozrin** [comment]
> first of all i am a big fan of laragon but today i tried herd and i love the ui bud sadly some must have futures are paid which will make me staying at laragon. What you guys think about Herd? Will you move to Herd or staying at laragon?

They are both about the same in my opinion. Herd is a little cleaner and has fewer quirks. Docker is definitely the way to go. Just finished evaluating Laragon and found it clunky and assumptive. The "final nail in the coffin" was ... after I uninstalled it .. Laragon left 16 registry entries and over 200 mb of files on my computer.

![image](https://github.com/leokhoa/laragon/assets/6646725/8384a221-0e62-4f3c-9d12-470c4524c499)

**Reply by litegral** [reply]
What tool is this?

**Reply by NukeZero** [reply]
Revo Uninstaller -> https://www.revouninstaller.com/
